lib/components/Actions/Actions.scss

```diff
@@ -0,0 +1,22 @@
+@import '../../theme/tokens/facade';
+
+.root {
+	$gap: var(--global--space--3);
+
+	width: 100%;
+	display: flex;
+	flex: 0 0 auto;
+	align-content: center;
+
+	&:not(.equalWidth) {
+		& > :not(:last-child) {
+			margin-right: $gap;
+		}
+	}
+
+	&.equalWidth {
+		display: grid;
+		grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
```

Comment:
Does this mean we can only have two buttons side by side before it breaks?
![Screen Shot 2019-07-23 at 4 17 46 pm](https://user-images.githubusercontent.com/1334204/61687362-97b47900-ad65-11e9-9a41-30d78119e5c0.png)